integration-tests/general/tests/src/test/kotlin/com/r3/conclave/integrationtests/general/tests/ThreadSafeEnclaveTests.kt

```diff
@@ -89,13 +91,21 @@ class ThreadSafeEnclaveTests : AbstractEnclaveActionTest("com.r3.conclave.integr
 
     @Test
     fun `threading inside enclave`() {
+        //  For Gramine, we want this test to run only in SIMULATION mode
+        // CON-1270: Requesting too many threads is failing in Gramine
+        if (TestUtils.RuntimeType.GRAMINE.name == TestUtils.runtimeType.name) {
```

Comment:
`RuntimeType` is an enum, so there's no need to compare the name Strings. Remove the `.name` calls.

Reply:
And swap the comparison around. `TestUtils.runtimeType == TestUtils.RuntimeType.GRAMINE` is easier and more expected.

Reply:
Done. Make sense :)